packages/perseus/src/perseus-types.ts

```diff
@@ -97,7 +97,7 @@ type Widget<Type extends string, Options> = {
     // NOTE: perseus_data.go says this is required even though it isn't necessary.
     alignment?: string;
     // Options specific to the type field of the widget.  See Perseus*WidgetOptions for more details
-    options: Options | null | undefined;
+    options: Options;
```

Comment:
I couldn't find where a widget would be missing options?

Reply:
I took a Quick Look at our English published content. Looks like there are two widget configs missing the `options` but they are also missing everything so I'd say they're invalid.  I'm in favour of making `options` non-optional. 👍